router/demo.go

```diff
@@ -0,0 +1,30 @@
+package router
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type demoController interface {
+	Fetch(w http.ResponseWriter, r *http.Request)
+	FetchById(w http.ResponseWriter, r *http.Request)
+	Feed(w http.ResponseWriter, r *http.Request)
+	UpdateUsersFromFeed(w http.ResponseWriter, r *http.Request)
+}
+
+func Setup(c demoController) *mux.Router {
```

Comment:
Set some comments, about exported functions, this is a good practice and is recommended so whenever other people would try to add new code or modify the current one, could get a context without checking all the project.

// Setup do..., expect... and will return... would be good enough but there are a lot of styles for this comments